### PR TITLE
Enable four new Style cops

### PR DESCRIPTION
**Pull request by kaylareopelle**
# Overview

Turns on four previously disabled cops: 
* Style/MultilineTernaryOperator
* Style/NilComparison
* Style/NonNilCheck
* Style/OrAssignment

### CODE REVIEW

**Review by fallwith — DISMISSED** [review]
I like all four of these.

Where did we land on being redundant with RuboCop's defaults? My desired approach would be to simply remove these 4 from `.rubocop.yml`, but I definitely feel stronger about having them enabled than _how_ they are enabled.

**Comment by kaylareopelle** [comment]
> I like all four of these.
> 
> Where did we land on being redundant with RuboCop's defaults? My desired approach would be to simply remove these 4 from `.rubocop.yml`, but I definitely feel stronger about having them enabled than _how_ they are enabled.

@fallwith  - I've updated the `.rubocop.yml` for this PR to reflect our team discussion!

File: .rubocop.yml [hunk 1]
```diff
@@ -1596,19 +1593,17 @@ Style/NestedTernaryOperator:
 Style/Next:
   Enabled: false
 
-# Disabling for now
 Style/NilComparison:
-  Enabled: false
-  # EnforcedStyle: predicate
+  Enabled: true
+  EnforcedStyle: predicate
```
Comment (fallwith) [inline .rubocop.yml:1598]: This cop's config is now redundant with [these defaults](https://github.com/rubocop/rubocop/blob/master/config/default.yml#L4366). We can remove all of these lines from .rubocop.yml

File: .rubocop.yml [hunk 2]
```diff
@@ -1596,19 +1593,17 @@ Style/NestedTernaryOperator:
...
 # Disabling for now
 Style/NilLambda:
   Enabled: false
 
-# Disabling for now
 Style/NonNilCheck:
-  Enabled: false
-  # IncludeSemanticChanges: false
+  Enabled: true
+  IncludeSemanticChanges: false
```
Comment (fallwith) [inline .rubocop.yml:1606]: This cop's config is now redundant with [these defaults](https://github.com/rubocop/rubocop/blob/master/config/default.yml#L4383) and can be removed entirely from .rubocop.yml.

File: .rubocop.yml (hunk 1 above)
Comment (kaylareopelle) [reply .rubocop.yml]: Nice catch! Resolved by: 347dcbc919d0c1d5ad64891dee9e39346af654aa

File: .rubocop.yml (hunk 2 above)
Comment (kaylareopelle) [reply .rubocop.yml]: Nice catch! Resolved by: 347dcbc919d0c1d5ad64891dee9e39346af654aa

**Review by fallwith — APPROVED** [review]

**Pull request merged**